### ISSUE TITLE
C4599 warning with Ninja

### ISSUE THREAD

**Issue by kazarmy** [kazarmy/cotire#6]
MSVC generates the following warning if the Ninja generator is used:

```
warning C4599: '/I<relative_path>': command line argument number 1 does not match
   precompiled header
```

**Comment by kazarmy** [comment]
Fixed by 9f772d5bd4a49e4cfc9537cdb307bb8cb8b3fd2e.

**Comment by kazarmy** [comment]
Related CMake issue is [issue #13894](https://gitlab.kitware.com/cmake/cmake/issues/13894).
